shub_image/utils.py

```diff
@@ -119,22 +119,24 @@ class CustomDockerClient(docker.Client):
 
         # XXX: workaround for https://github.com/docker/docker-py/issues/1059
         def _stream_helper(self, response, decode=False):
-            it = super(CustomDockerClient, self)._stream_helper(response, decode=decode)
+            it = super(CustomDockerClient, self)._stream_helper(
+                response, decode=decode)
             for data in it:
                 if not isinstance(data, string_types):
                     yield data
-                for line in data.split('\r\n'):
-                    line = line.strip()
-                    if line:
-                        yield line
+                else:
```

Comment:
I think we can actually drop this workaround since issues was fixed in docker-py 1.10 https://github.com/docker/docker-py/issues/1059#issuecomment-246043470